File: python/PROBLEMS/1158.py
n = int(input())
for i in range(0, n):
    x, y = list(map(int, input().split(' ')))
    cont = 0
    for j in range(0, y*2):
        if x%2 != 0:
            cont+=x
        x+=1
    print(cont)

File: c/PROBLEMS/1158.c
#include <stdio.h>

int main(){
    int n, i, x, y;
    scanf("%d", &n);

    for (i=0;i<n;i++){
        int cont=0;
        scanf("%d %d", &x, &y);
        for (int j=0;j<y*2;j++){
            if (x%2 != 0){
                cont+=x;
            }
            x++;
        }
        printf("%d\n", cont);
    }

}